apps/experiments/migrations/0035_alter_syntheticvoice_language_code_and_more.py

```diff
@@ -0,0 +1,25 @@
+# Generated by Django 4.2 on 2023-10-11 15:30
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
```

Comment:
this can be combined with migration 0033 (just delete both and re-run `makemigrations`)

Reply:
I didn't realise the rebase broke the migrations as well. I think I've fixed it all in #a29797e

Reply:
(it's still broken because `main` changed again. Fixed in https://github.com/dimagi/open-chat-studio/pull/48/commits/2ffb9f64f08378d402aa442c68c2a309562e71a4